docs/tutorials/basic_commands.md

```diff
@@ -1,23 +1,19 @@
 # Getting Started with Ubuntu Pro Client
 
-The Ubuntu Pro Client (`pro`) provides users with a simple mechanism to
-view, enable, and disable offerings from Canonical on their system. In this tutorial
-we will cover the base `pro` commands that allow a user to successfully manage the offering
-on their machine.
+The Ubuntu Pro Client (`pro`) provides a simple mechanism for viewing, enabling and
+disabling Canonical offerings on your system. In this tutorial we will cover the base
+`pro` commands that will allow you to successfully manage the offerings on your machine.
```

Comment:
I think here we might want to be a little more specific in saying exactly what the aim of the tutorial is (i.e., what we are going to guide them through), which can then be echoed again in a "summing up" at the very end to reinforce what the user has accomplished. Since tutorials are for learning, I think it's nice to have these sorts of things explicitly stated for the purpose of cementing them in the mind of the user through a sort of repetition/echo (also, if they make it to the end, it helps to leave them with a sense of achievement through completing the steps successfully).

Reply:
I agree, I think that reads in a much more friendly way - thumbs up from me!